core/src/main/java/nl/nn/adapterframework/stream/Message.java

```diff
@@ -709,6 +711,12 @@ public static Message asMessage(Object object) {
 		if (object instanceof Path) {
 			return new PathMessage((Path) object);
 		}
+		if (object instanceof MessageWrapper) {
+			return ((MessageWrapper<?>)object).getMessage();
+		}
+		if (object instanceof RawMessageWrapper) {
+			return new Message(null, ((RawMessageWrapper<?>)object).getRawMessage());
+		}
```

Comment:
Dit kan ook mooier?

Reply:
Nee. Ik doe nu een `throw` in deze case want ik denk eigenlijk dat je dit niet moet mogen doen.